controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -338,6 +338,8 @@ func newRookCephOperatorConfig(namespace string) *corev1.ConfigMap {
 	data["CSI_PROVISIONER_TOLERATIONS"] = defaultCSIToleration
 	data["CSI_PLUGIN_TOLERATIONS"] = defaultCSIToleration
 	data["CSI_LOG_LEVEL"] = "5"
+	data["CSI_ENABLE_OMAP_GENERATOR"] = "true"
+	data["CSI_ENABLE_VOLUME_REPLICATION"] = "true"
```

Comment:
@Madhu-1 @leseb @ShyamsundarR  I am even enabling these by default. Let me know if it isn't a good idea.

Reply:
IMo this should be set only if `mirroring.enabled` is `true`.

Reply:
But, it is enabled by default!!

Reply:
what if the user chooses to change the mirroring.enabled to `false` and also from the upgrade point of view looks like `ensureRookCephOperatorConfig` will not update the configmap if the configmap already exists? what about the upgrade path?

Reply:
ahh I forgot about the upgrade path. We need to patch config whenever mirroring is enabled.
Thanks for pointing it out.

Reply:
It's still not clear how we handle operator config. Dropped these changes for now.
It seems this affects more than just pool mirroring.